webapp/components/product-info.js

```diff
@@ -45,7 +46,7 @@ const versionPatterns = Object.freeze({
 
 // The set of all browsers known to the wpt.fyi UI.
 const AllBrowserNames = Object.freeze(['android_webview', 'chrome_android', 'chrome_ios', 'chrome',
-  'deno', 'edge', 'firefox', 'flow', 'node.js', 'safari', 'servo', 'webkitgtk']);
+  'deno', 'edge', 'firefox', 'flow', 'node.js', 'safari', 'servo', 'webkit', 'webkitgtk']);
```

Comment:
```suggestion
  'deno', 'edge', 'firefox', 'flow', 'node.js', 'safari', 'servo', 'webkitgtk', 'webkit']);
```

I don't know why, but rearranging these made the test pass for me. Maybe someone smarter than me can figure out why. 😅